src/lib/litegraph/src/node/NodeSlot.ts

```diff
@@ -33,6 +33,7 @@ export interface IDrawOptions {
 /** Shared base class for {@link LGraphNode} input and output slots. */
 export abstract class NodeSlot extends SlotBase implements INodeSlot {
   pos?: Point
+  rotation = -Math.PI / 2
```

Comment:
If this is a constant, can it be either a `readonly static` or a `CONSTANT` in the module scope?